Gemfile

```diff
@@ -0,0 +1,78 @@
+source "https://rubygems.org"
```

Comment:
please explain every line of this file to me next wednesday

Reply:
Sure